### DISCUSSION TITLE
HDD's not detected

### DISCUSSION THREAD

**Discussion by Olddantrucker** [86Box/86Box#1511] [Q&A] [answered]
Okay I managed to get my hard drives detected but it's saying no boot disc found even though I have win 95 boot disc inserted into floppy drive.  I changed the order between C A to A C and still no dice.  I was going to delete this thread  and start a different one but there is no option so I edited most of my post.

**Comment by Olddantrucker** [comment, marked as answer]
Turns out I was using the wrong version of 86box. I used the 'recommended' version and decided to try one called Ken and it works. Though I could only use S3 Virge as other graphic drivers all fail.  I also had to install SB128PCI manually from Creative's website as the built in sound detection would never find it saying an 'error' occurred but manually installing fixed it.  Too bad the options don't always do what they say.  I wish I could delete this whole post though on the other hand there should be some kind of warning for people out there to use an 'optimized' version of 86box. https://support.creative.com/downloads/download.aspx?nDownloadId=1843 SB128 Drivers for Win9X. 

Oh and the optimized table page leads to an error page. http://ci.86box.net/job/86Box-Optimized/
<html>
<body>
<h2 style="color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">HTTP ERROR 404 Not Found</h2>

URI: | /job/86Box-Optimized/
-- | --
404
Not Found
Stapler

<br class="Apple-interchange-newline">
</body>
</html>

**Reply by bachimusprime** [reply]
The reason there's no warning for those using optimized builds of 86Box is because we discontinued them months ago. Use the builds just labelled "86Box" at ci.86box.net for the best experience.
